eng/scripts/Stop-TestProxy.ps1

```diff
@@ -0,0 +1 @@
+﻿Stop-Process -Name "test-proxy"
```

Comment:
```suggestion
# Copyright (c) Microsoft Corporation. All rights reserved.
# SPDX-License-Identifier: MIT

﻿Stop-Process -Name "test-proxy"
```

Reply:
I was going to give the same feedback, but turns out majority (if not all) of the scripts in eng/scripts lack the license header.

I was told it wasn't necessary to add them there.

cc @danieljurek, @weshaggard 

Same in other repos like Embedded C:
https://github.com/Azure/azure-sdk-for-c/pull/2304#discussion_r943813969